ministryofjustice/correspondence_tool_staff
# CT-2962 update exempt and final pagecounts

## Description
Adds ability to update the exempt and final page counts on the app.

## Notes for reviewer
- do the tests seem comprehensive enough? Note I want to get this in and then will work on cleaning up the spec to use a PageObject at a later date.
- Do the validations in the `Offender.rb` make sense to you?
- also note that I haven't moved the record data request button as per the design, as it falls into another PR.


## Self-review checklist


### Screenshots
**Before:**
_**Data requested table:**_

<img width="666" alt="Screenshot 2020-08-19 at 15 05 13" src="https://user-images.githubusercontent.com/13377553/90645129-7efae900-e22d-11ea-9a49-fe3517108229.png">

**After with new screens:**

_**Data requested table:**_
 
<img width="679" alt="Screenshot 2020-08-19 at 15 05 30" src="https://user-images.githubusercontent.com/13377553/90645156-87ebba80-e22d-11ea-845e-4ed06022b51f.png">

_**Exempt pages form:**_
<img width="658" alt="Screenshot 2020-08-19 at 15 08 15" src="https://user-images.githubusercontent.com/13377553/90645518-efa20580-e22d-11ea-8575-0ebb136df483.png">

_**Pages for dispatch form:**_
<img width="657" alt="Screenshot 2020-08-19 at 15 08 49" src="https://user-images.githubusercontent.com/13377553/90645534-f4ff5000-e22d-11ea-8dfc-99a3203ef6db.png">


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2962

### Manual testing instructions
1. Log in as Branston user
2. Go to case details page
3. Click "Update exempt pages"
4. Enter whole number figure, click "Continue"
5. See new figure reflected in blue box
6. Repeat steps 4-5 after clicking "Update pages for dispatch"
7. Are both figures reflected in blue box?
